src/plays/savings-calculator/SavingsCalculator.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+import Total from "./components/Total";
+import SavingOptions from "./components/SavingOptions";
+import "./styles/savingsCalculator.scss";
+
+function SavingsCalculator() {
+  const [total, setTotal] = useState(2327);
+  const [currency, setCurrency] = useState("INR");
+
+  return (
+    <div className="savingsCalculator__container">
+      <div className="savingsCalculator">
+        <Total total={total} currency={currency} />
+        <SavingOptions
+          currency={currency}
+          setCurrency={setCurrency}
+          setTotal={setTotal}
+        />
+      </div>
+    </div>
+  );
+}
```

Comment:
@kd100100 , Can we follow the play structure the same as here?
https://github.com/reactplay/react-play/blob/main/src/plays/clock/CurrentTimer.jsx

Reply:
Added play's header!